[info] Running com.glyph.Main 
[error] Exception in thread "LWJGL Application" com.badlogic.gdx.utils.GdxRuntimeException: com.badlogic.gdx.utils.GdxRuntimeException: Couldn't load dependencies of asset 'data/card1.png'
[error] 	at com.badlogic.gdx.assets.AssetManager.handleTaskError(AssetManager.java:497)
[error] 	at com.badlogic.gdx.assets.AssetManager.update(AssetManager.java:337)
[error] 	at com.glyph.libgdx.Director.render(Director.java:33)
[error] 	at com.glyph.libgdx.Engine.render(Engine.java:36)
[error] 	at com.badlogic.gdx.backends.lwjgl.LwjglApplication.mainLoop(LwjglApplication.java:202)
[error] 	at com.badlogic.gdx.backends.lwjgl.LwjglApplication$1.run(LwjglApplication.java:110)
[error] Caused by: com.badlogic.gdx.utils.GdxRuntimeException: Couldn't load dependencies of asset 'data/card1.png'
[error] 	at com.badlogic.gdx.assets.AssetLoadingTask.handleAsyncLoader(AssetLoadingTask.java:126)
[error] 	at com.badlogic.gdx.assets.AssetLoadingTask.update(AssetLoadingTask.java:94)
[error] 	at com.badlogic.gdx.assets.AssetManager.updateTask(AssetManager.java:434)
[error] 	at com.badlogic.gdx.assets.AssetManager.update(AssetManager.java:335)
[error] 	... 4 more
[error] Caused by: java.util.concurrent.ExecutionException: com.badlogic.gdx.utils.GdxRuntimeException: Couldn't load file: data/card1.png
[error] 	at java.util.concurrent.FutureTask$Sync.innerGet(FutureTask.java:222)
[error] 	at java.util.concurrent.FutureTask.get(FutureTask.java:83)
[error] 	at com.badlogic.gdx.assets.AssetLoadingTask.handleAsyncLoader(AssetLoadingTask.java:124)
[error] 	... 7 more
[error] Caused by: com.badlogic.gdx.utils.GdxRuntimeException: Couldn't load file: data/card1.png
[error] 	at com.badlogic.gdx.graphics.Pixmap.<init>(Pixmap.java:140)
[error] 	at com.badlogic.gdx.assets.loaders.TextureLoader.loadAsync(TextureLoader.java:66)
[error] 	at com.badlogic.gdx.assets.loaders.TextureLoader.loadAsync(TextureLoader.java:39)
[error] 	at com.badlogic.gdx.assets.AssetLoadingTask.call(AssetLoadingTask.java:74)
[error] 	at com.badlogic.gdx.assets.AssetLoadingTask.call(AssetLoadingTask.java:36)
[error] 	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
[error] 	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
[error] 	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
[error] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
[error] 	at java.lang.Thread.run(Thread.java:662)
[error] Caused by: com.badlogic.gdx.utils.GdxRuntimeException: File not found: data\card1.png (Internal)
[error] 	at com.badlogic.gdx.files.FileHandle.read(FileHandle.java:132)
[error] 	at com.badlogic.gdx.files.FileHandle.length(FileHandle.java:585)
[error] 	at com.badlogic.gdx.files.FileHandle.readBytes(FileHandle.java:220)
[error] 	at com.badlogic.gdx.graphics.Pixmap.<init>(Pixmap.java:137)
[error] 	... 9 more
